public/src/app/comment-modal/comment-modal.component.html

```diff
@@ -19,8 +19,8 @@ <h4 class="modal-title">Submit a Comment</h4>
 
   <div class="modal-body">
     <h5>Terms and Conditions</h5>
-    <p>Personal information is collected under the authority section 26(c) of the Freedom of Information and Protection of Privacy Act. Personal information includes but is not limited to your opinion, name, phone number, email and mailing address. The personal information collected is directly related to the processing of Crown land tenure applications and may be considered by the Ministry of Forests, Lands, Natural Resource Operations and Rural Development further to the administration of the Land Act.</p>
-    <p>If you have any questions about the collection, use and disclosure of the information collected, contact the Manager, Policy and Land Operations Support at the Ministry of Forests, Lands, Natural Resource Operations and Rural Development.</p>
+    <p>Personal information is collected under the authority section 26(c) of the Freedom of Information and Protection of Privacy Act. Personal information includes but is not limited to your opinion, name, phone number, email and mailing address. The personal information collected is directly related to the processing of Crown land tenure applications and may be considered by the Ministry of Land, Water and Resource Stewardship further to the administration of the Land Act.</p>
+    <p>If you have any questions about the collection, use and disclosure of the information collected, contact the Manager, Policy and Land Operations Support at the Ministry of Land, Water and Resource Stewardship.</p>
```

Comment:
Issue #46 requests the ministry be renamed to "Ministry of Forests". Maybe there's discussion I missed on using LWRS instead, but I just wanted to verify.

Reply:
Crap, you're right!  Fixed.